application/src/main/kotlin/net/corda/v5/application/crypto/DigitalSignatureVerificationService.kt

```diff
@@ -1,25 +1,30 @@
 package net.corda.v5.application.crypto
 
+import net.corda.v5.base.annotations.DoNotImplement
 import net.corda.v5.crypto.SignatureSpec
 import java.security.InvalidKeyException
 import java.security.PublicKey
 import java.security.SignatureException
 
 /**
- * The [DigitalSignatureVerificationService] digital signature verification operations.
+ * [DigitalSignatureVerificationService] allows flows to verify digital signatures.
+ * The platform will provide an instance of [DigitalSignatureVerificationService] to flows via property injection.
```

Comment:
Think the line about property injection should have a blank line between it and the rest of the content. Same for the others.